apiary.apib

```diff
@@ -2158,6 +2159,10 @@ Get a list of quotations.
                     + purchase_price (Money, nullable)
                 + created_at: `2017-05-09T11:25:11+00:00` (string, nullable)
                 + updated_at: `2017-05-09T11:30:58+00:00` (string, nullable)
+                + status: `open` (enum)
+                    + Members
```

Comment:
Should we make this future proof and add 'rejected' and closed as well?
@stivni wdyt?

Reply:
closed and rejected were added.